jest/setup.js

```diff
@@ -105,6 +105,12 @@ const mockNativeModules = {
   FbRelayNativeAdapter: {
     updateCLC: jest.fn(),
   },
+  Geolocation: {
+    getCurrentPosition: jest.fn(),
+    watchPosition: jest.fn(),
+    clearWatch: jest.fn(),
```

Comment:
Instead of mocking `watchPosition` and `clearWatch`, can you just mock `startObserving()`? We should only need to mock the native APIs here.